common/debug.h

```diff
@@ -71,13 +71,13 @@ void              p11_debug_precond             (const char *format,
 #endif
 
 #define return_val_if_fail(x, v) \
-	do { if (!(x)) { \
+	do { if (x) { } else {							\
```

Comment:
Seems ok but ugly. Is it necessary to address such false-positives?

Reply:
Yes, the same pattern can be found in the corresponding glib macros.